package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "relay-pagination-directive",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
this change shouldn't be here, as the version in master already has this value

Reply:
Rebased master to pick up this change